components/flex/index.en-US.md

```diff
@@ -0,0 +1,36 @@
+---
+category: Components
+group: Layout
+title: Flex
+cover: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*oyOcTrB12_YAAAAAAAAAAAAADrJ8AQ/original
+coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*M7_ER7GJr6wAAAAAAAAAAAAADrJ8AQ/original
+demo:
+  cols: 2
+tag: New
+---
+
+Flex. Available since `5.9.0`.
+
+## When To Use
+
+- Show evaluation.
+- A quick rating operation on something.
```

Comment:
These lines are still related to Rating component

Reply:
I have made corrections, thx.